app/vmselect/netstorage/tmp_blocks_file.go

```diff
@@ -14,6 +15,10 @@ import (
 	"github.com/VictoriaMetrics/metrics"
 )
 
+var (
+	tmpBufSize = flag.Int("search.tmpBufSize", 4194304, "In large memory scenarios, controlling the size of temporary buffers to use memory more efficiently")
```

Comment:
It's better to use flagutil.NewBytes(). It allows to configure human-readable values.

Reply:
Also, flag name search.tmpBufSizeBytes looks better to me.